dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers.test.ts

```diff
@@ -51,10 +63,38 @@ describe('get-utilisation-report-reconciliation-details-by-id.controller helpers
       });
     });
 
-    it('maps the fee record entity to the reconciliation details fee record item when the currencies do not match', () => {
+    it('converts the fees paid to ukef for the period to the reported fees when the payment currencies match', () => {
+      // Arrange
+      const feesPaidToUkefForThePeriodCurrency: Currency = 'GBP';
+      const feesPaidToUkefForThePeriod = 100.0;
+      const paymentCurrency: Currency = 'GBP';
+
+      const feeRecordEntity = FeeRecordEntityMockBuilder.forReport(uploadedReport)
+        .withFeesPaidToUkefForThePeriodCurrency(feesPaidToUkefForThePeriodCurrency)
+        .withFeesPaidToUkefForThePeriod(feesPaidToUkefForThePeriod)
+        .withPaymentCurrency(paymentCurrency)
+        .build();
+
+      // Act
+      const feeRecordItem = mapFeeRecordEntityToReconciliationDetailsFeeRecordItem(feeRecordEntity);
+
+      // Assert
+      expect(feeRecordItem.reportedFees).toEqual<CurrencyAndAmount>({
+        currency: feesPaidToUkefForThePeriodCurrency,
+        amount: feesPaidToUkefForThePeriod,
+      });
+      expect(feeRecordItem.reportedPayments).toEqual<CurrencyAndAmount>({
+        currency: paymentCurrency,
+        amount: feesPaidToUkefForThePeriod,
+      });
+      expect(feeRecordItem.totalReportedPayments).toEqual<CurrencyAndAmount>({
+        currency: paymentCurrency,
+        amount: feesPaidToUkefForThePeriod,
+      });
```

Comment:
So your test name says you are checking it maps the 'fees paid' to 'reported fees', which is the first assertion, but then you have these two assertions in addition. These should either be removed or if they are important in a different test.